lib/minify.js

```diff
@@ -146,6 +147,9 @@ function minify(files, options) {
             result.warnings = warnings;
         }
         return result;
+    } catch (ex) {
+        if (options.catch) return ex;
```

Comment:
I've picked `error` and amended the commit in https://github.com/mishoo/UglifyJS2/pull/1880/commits/0959150df491edf93a0d72c3de7801b157bb9efc